nixos/modules/services/monitoring/prometheus/exporters/unbound.nix

```diff
@@ -0,0 +1,54 @@
+{ config, lib, pkgs, options }:
+
+with lib;
+
+let
+  cfg = config.services.prometheus.exporters.unbound;
+in
+{
+  port = 9167;
+  extraOpts = {
+    fetchType = mkOption {
+      type = types.enum [ "shm" "tcp" "uds" ];
```

Comment:
Hmm, I see that there are also parameters missing from the module for `tcp` mode, which are as essential for successful startup as `--control-interface` for `uds` mode.
At least `--shm-key` defaults to 11777.

Reply:
> I'd prefer to remove "shm" from this enum. Otherwise the --shm-key parameter should also be exposed by the module, ideally with an assertion that a key for the memory segment is specified

Makes sense. Removed for now.